arangod/Cluster/SynchronizeShard.cpp

```diff
@@ -1073,8 +1056,8 @@ ResultT<TRI_voc_tick_t> SynchronizeShard::catchupWithReadLock(
     }
     lastLogTick = tickReached;
     if (didTimeout) {
-      LOG_TOPIC("e516e", INFO, Logger::MAINTENANCE)
-          << "Renewing softLock for " << shard << " on leader: " << leader;
+      LOG_TOPIC("7531f", INFO, Logger::MAINTENANCE)
```

Comment:
We change this LOG id?

Reply:
my bad. will fix back. had to go through some merge conflicts given the age of the PR.